include/javascript/mozaik/jquery.mozaik.js

```diff
@@ -46,7 +46,7 @@ var mozaik = {
     getMozaikInnerHTML: function(name, innertext, style) {
         //style = ($(innertext).attr('style') ? $(innertext).attr('style') + ';' : '') + style;
         //style = style.replace(/;\s*;/, ';');
-        var html = '<div class="mozaik-inner"' + (name ? ' data-name="' + name + '"' : '') + ' style="' + style + '">' + innertext + '</div>';
+        var html = '<div class="mozaik-inner"' + (name ? ' data-name="' + name + '"' : '') + ' style="background-color: #fafafa; ' + style + '">' + innertext + '</div>';
```

Comment:
It would be better if you implemented this as css class and then added a variable to variables.css. That way it can be configured.